server/models/dataset_helpers/image_metadata.py

```diff
@@ -720,7 +720,11 @@ def load(cls, value, acquisition, clinical, private):
 def _populateMetadata(acquisition, clinical):
     """
     Populate empty metadata fields that can be determined based on other fields.
+    In some cases, populates inconsistent fields and emits a warning.
+    Returns a list of warnings.
     """
+    warnings = []
```

Comment:
In the future (I think it's out of scope of this PR), we could consider using the [built-in `warnings` library](https://docs.python.org/2.7/library/warnings.html) for this.

Reply:
I'm not clear how that library is appropriate for an application-level list of strings that happen to be warnings about the data.

Reply:
I think we'd use a [`warnings.catch_warnings`](https://docs.python.org/2.7/library/warnings.html#warnings.catch_warnings) context manager around our application logic. I'm not certain this is ultimately a good idea, just that we might want to consider it (so we don't have to keep passing and returning `warnings` variables).

Reply:
Ah, I see. I agree that catch_warnings, or a similar construct, would improve the maintainability of this code. Here's an example of what that might look like: ://docs.python.org/3/library/warnings.html#testing-warnings